tools/scripts/goimports/main.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// defaultArgs are always passed to goimports.
+// -w: write result to (source) file instead of stdout
```

Comment:
OPTIONAL NIT: Replace `-w` with `-i` so its more similar to `sed`'s "in place" flag.

Reply:
This is not something we're specifying, these are specified by, and passed to, the `goimports` binary.